sys/net/gnrc/routing/rpl/gnrc_rpl_control_messages.c

```diff
@@ -729,32 +729,55 @@ void gnrc_rpl_send_DAO(gnrc_rpl_instance_t *inst, ipv6_addr_t *destination, uint
         return;
     }
 
-    gnrc_ipv6_netif_addr_t *me_netif = gnrc_ipv6_netif_addr_get(me);
-    if (me_netif == NULL) {
-        DEBUG("RPL: no netif address found for %s\n",
-              ipv6_addr_to_str(addr_str, me, sizeof(addr_str)));
-        return;
+    fib_entry_t *fentry = NULL;
+    ipv6_addr_t *addr = NULL;
+
+    mutex_lock(&(gnrc_ipv6_fib_table.mtx_access));
```

Comment:
I feared the moment where the FIB tables are exposed and its entries are changed outside the FIB